codec-redis/src/main/java/io/netty/handler/codec/redis/RedisMessageType.java

```diff
@@ -23,28 +23,35 @@
 @UnstableApi
 public enum RedisMessageType {
 
+    INLINE_COMMAND(null, true),
     SIMPLE_STRING((byte) '+', true),
     ERROR((byte) '-', true),
     INTEGER((byte) ':', true),
     BULK_STRING((byte) '$', false),
-    ARRAY_HEADER((byte) '*', false),
-    ARRAY((byte) '*', false); // for aggregated
+    ARRAY_HEADER((byte) '*', false);
 
-    private final byte value;
+    private final Byte value;
     private final boolean inline;
 
-    RedisMessageType(byte value, boolean inline) {
+    RedisMessageType(Byte value, boolean inline) {
         this.value = value;
         this.inline = inline;
     }
 
     /**
-     * Returns prefix {@code byte} for this type.
+     * Returns prefix {@code byte} for this type or {@code null} if its length is zero.
      */
-    public byte value() {
+    public Byte value() {
         return value;
     }
 
+    /**
+     * Returns length of this type.
+     */
+    public int length() {
+        return (value != null) ? RedisConstants.TYPE_LENGTH : 0;
```

Comment:
nit: remove parenthesis around condition for consistency with other ternaries, for ex. in `RedisDecoder`

Reply:
Fixed!